charts/observability-pipelines-worker/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.6.0
+
+* Official image `1.6.0`
+* Remove the deprecated `region` option for the Datadog components
+* Add the socket source
```

Comment:
```suggestion
```

Looking at the previous releases, it doesn't look like we call out OPW changes here, just chart changes; which makes sense to me since we should have a separate changelog for OPW.

Reply:
ok, I'll updated 1.5.1 as well to remove the OPW change callout there.